Makefile

```diff
@@ -92,8 +96,13 @@ install:
 	install -m 644 $(wildcard docs/*.8.md) $(PREFIX)/share/man/man8
 	install -d $(PREFIX)/share/man/man5
 	install -m 644 $(wildcard docs/*.5.md) $(PREFIX)/share/man/man5
+	install -D -m 644 ocid.service $(PREFIX)/lib/systemd/system
+	install -D -m 644 ocid.conf $(DESTDIR)/etc
 
 uninstall:
+	systemctl stop ocid.service
+	systemctl disable ocid.service
```

Comment:
I actually don't really like that we're installing the `service` file on `make install`. But we can deal with that later.